cvat/apps/documentation/faq.md

```diff
@@ -0,0 +1,81 @@
+# Frequently asked questions
+- [How to update CVAT](#how-to-update-cvat)
+- [Kibana app works, but no logs here](#kibana-app-works-but-no-logs-here)
+- [How to change default CVAT hostname or port](#how-to-change-default-cvat-hostname-or-port)
+- [How to configure connected share folder on Windows](#how-to-configure-connected-share-folder-on-windows)
+- [How to make unassigned tasks not be visible to all users](#how-to-make-unassigned-tasks-not-be-visible-to-all-users)
+- [Can Nvidia GPU be used to perform inference with my own model](#can-nvidia-gpu-be-used-to-perform-inference-with-my-own-model)
+
+## How to update CVAT
+Before upgrading, please follow the oficcial docker
```

Comment:
@azhavoro, Could you please check obvious typos by a spell checker? For example, `oficcial`. Please ask @aschernov help as well.